php/commands/search-replace.php

```diff
@@ -61,6 +62,9 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * : Write transformed data as SQL file instead of saving replacements to
 	 * the database. If <file> is not supplied, will output to STDOUT.
 	 *
+	 * [--export_insert_size[=<rows>]]
+	 * : Define number of rows in single INSERT statement when doing SQL export.
```

Comment:
Can you add a short explanation as to why you might use or change this value?

Also, to specify the default value in the document, you can use inline YAML doc:

```
---
default: 50
---
```